exec/csnex/Extensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace csnex
+{
+    internal static partial class CodeExtensions
+    {
+        public static int Compare(this byte[] self, byte[] rhs)
```

Comment:
Why is this function written this way, as a method? Shouldn't it just be an ordinary function that takes two `byte[]` parameters and returns an `int` result? Not everything needs to be a method.